public/app/features/admin/UpgradePage.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { PureComponent } from 'react';
+import { NavModel } from '@grafana/data';
+import Page from '../../core/components/Page/Page';
+import { LicenseChrome, Orbit } from './LicenseChrome';
+import { Button } from '@grafana/ui';
+import { hot } from 'react-hot-loader';
+import { StoreState } from '../../types';
+import { getNavModel } from '../../core/selectors/navModel';
+import { connect } from 'react-redux';
+
+interface Props {
+  navModel: NavModel;
+}
+
+export class UpgradePage extends PureComponent<Props> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  async componentDidMount() {}
+
+  render() {
+    const { navModel } = this.props;
+
+    return (
+      <Page navModel={navModel}>
+        <Page.Contents>
+          <div className="page-container page-body">
+            <UpgradeInfo />
+          </div>
+        </Page.Contents>
+      </Page>
+    );
+  }
+}
+
+const title = { fontWeight: 600, fontSize: '26px', lineHeight: '123%' };
+
+export class Experiments {
+  LargerCTA?: boolean;
+  HeaderCTA?: boolean;
+}
+
+export const ExperimentsContext = React.createContext(new Experiments());
+
+export function UpgradeInfo() {
+  const experiments = React.useContext(ExperimentsContext);
+  const subheader = experiments.HeaderCTA ? (
+    <CallToAction larger={experiments.LargerCTA} />
+  ) : (
+    <h3>Get your free trial</h3>
+  );
+
+  return (
+    <React.Fragment>
```

Comment:
You can simply use 
```suggestion
    <>
    </>
```
istead of 
```tsx
<React.Fragment>
...
</React.Fragment>